adafruit_esp32spi/adafruit_esp32spi_wifimanager.py

```diff
@@ -202,6 +202,16 @@ def ping(self, host, ttl=250):
         self.pixel_status(0)
         return response_time
 
+    def ip_address(self):
+        """
+        Returns a formatted local IP address, update status pixel.
+        """
+        if not self._esp.is_connected:
+            self.connect()
```

Comment:
I'm kinda 🤷‍♂️ about things like this while handy do something other than what was requested. I would personally return an error here but this is definitely more convenient.

Reply:
I missed that this is how most of the lib works; no change needed here!